src/pages/edge/adobe-journey-optimizer-decisioning/tabs/api-reference.md

```diff
@@ -336,7 +468,59 @@ Optimize.updatePropositions(decisionScopes,
                             });
 ```
 
-<Variant platform="android" api="update-propositions-withError" repeat="6"/>
```

Comment:
can you please check if this is referenced from '/adobe-journey-optimizer-decisioning/api-reference.md', please rename there if linking is broken.

Reply:
updated the reference as well @ashumadobe